src/main/resources/application.properties

```diff
@@ -28,3 +28,5 @@ spring.mvc.format.date-time=iso
 
 app.startQtrYYYYQ=${START_QTR:${env.START_QTR:20221}}
 app.endQtrYYYYQ=${END_QTR:${env.END_QTR:20222}}
+
+app.ucsb.api.consumer_key=${UCSB_API_KEY:${env.UCSB_API_KEY:see-instructions-in-readme}}
```

Comment:
Is is supposed to be `see-instructions-in-readme` here?

Reply:
Yes